_episodes_rmd/11-writing-data.Rmd

```diff
@@ -52,6 +52,7 @@ using the arguments to this function.
 pdf("Life_Exp_vs_time.pdf", width=12, height=4)
 ggplot(data=gapminder, aes(x=year, y=lifeExp, colour=country)) +
   geom_line()
+theme(legend.position="none")
```

Comment:
Good to follow general style guidelines with your additions, even if the existing content isn't all fixed up yet. We follow both the [Google R Style Guide](https://google.github.io/styleguide/Rguide.xml) and [Hadley's recommendations from Advanced R](http://adv-r.had.co.nz/Style.html), so you should indent the `theme()` line, and add spaces around operators line `=`. Also the `+` should be at the end of the line so that the first two lines of the `ggplot()` call aren't run without the last one.